commands

```diff
@@ -131,7 +131,7 @@ EOF
 
 add_shared_key() {
   check_install
-  if [[ ! -n "$2" ]]; then
+  if [[ -n "$SHARED_KEY_TO_ADD" ]]; then
```

Comment:
where is this variable from?

Reply:
`SHARED_KEY_TO_ADD` was already defined at the really beginning of the script (line #5), where positional parameters are stored in variables.

I think issue #30 was caused by commit e977edd8367c1dec1f42f832900e676452f64262, where positional parameters were passed to `add_shared_key`, avoiding the condition `[[ ! -n "$2" ]]` to be fulfilled.

Reply:
Well what I think should happen is that the passed parameters should be checked correctly, not that we should use some weird global variables.

Reply:
You're absolutely right. I'll rewrite it this way.

Reply:
It should be better now, and the PR description has been updated.